src/adapters/scene-stream-access-manager.ts

```diff
@@ -95,13 +95,31 @@ export async function createSceneStreamAccessManagerComponent({
     return result.rowCount > 0 && result.rows[0].streaming
   }
 
+  async function getActiveStreamings(): Promise<Pick<SceneStreamAccess, 'created_at' | 'ingress_id'>[]> {
+    const result = await database.query<Pick<SceneStreamAccess, 'created_at' | 'ingress_id'>>(
+      SQL`SELECT created_at, ingress_id FROM scene_stream_access WHERE active = true AND streaming = true`
```

Comment:
I was thinking that instead of retrieving all active streams, we could get only the expired ones:
```suggestion
      SQL`SELECT created_at, ingress_id FROM scene_stream_access WHERE active = true AND streaming = true AND ${Date.now()} - created_at >  ${FOUR_HOURS}`
```
We can also limit the query to avoid having to retrieve a ton of data from the DB at once:
```suggestion
      SQL`SELECT created_at, ingress_id FROM scene_stream_access WHERE active = true AND streaming = true AND ${Date.now()} - created_at >  ${FOUR_HOURS} LIMIT 100`
```
We can even order it if we want to start with the older ones:
```suggestion
      SQL`SELECT created_at, ingress_id FROM scene_stream_access WHERE active = true AND streaming = true AND ${Date.now()} - created_at >  ${FOUR_HOURS} ORDER BY created_at DESC`
```

Reply:
good suggestions, thanks!